src/game/protos/object.h

```diff
@@ -240,6 +240,7 @@ int object_get_pal_limit(const object *obj);
 
 vec2i object_get_size(const object *obj);
 vec2i object_get_pos(const object *obj);
```

Comment:
object_get_pos still returns a vec2i.
it should probably be updated to return a vec2f.